randovania/games/prime2/exporter/joke_hints.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+JOKE_HINTS = [
+    "By this point in your run, you should have consumed at least 200 mL of water to maintain optimum hydration.",
```

Comment:
instead of duplicating, i would probably just do `JOKE_HINTS = generic_joke_hints + [ # echoes joke hints]`